Firestore/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # v9.0.0
+- [fixed] Fixed build errors in Xcode 13.3.1 (#9702).
```

Comment:
Fixed cmake build errors

Reply:
Reworked in [a3b8d8a](https://github.com/firebase/firebase-ios-sdk/pull/9747/commits/a3b8d8a654a5663849259ad254c53d63673d8816)